prow/config.yaml

```diff
@@ -868,17 +868,85 @@ presubmits:
           path: /mnt/disks/ssd0
 
   - name: pull-kubernetes-unit
-    agent: jenkins
+    agent: kubernetes
     always_run: true
     context: pull-kubernetes-unit
     rerun_command: "/test pull-kubernetes-unit"
     trigger: "(?m)^/test( all| pull-kubernetes-unit),?(\\s+|$)"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kube-test
+        args:
+        - "--clean"
+        - "--root=/go/src"
+        - "--job=$(JOB_NAME)"
+        - "--repo=github.com/Huawei-PaaS/$(REPO_NAME)=$(PULL_REFS)"
```

Comment:
this should just be `- "--repo=k8s.io/kubernetes=$(PULL_REFS)"` or `- "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"`